contracts/axelar-gateway/src/auth.rs

```diff
@@ -202,3 +199,9 @@ fn validate_signers(env: &Env, weighted_signers: &WeightedSigners) {
         panic_with_error!(env, AuthError::InvalidThreshold);
     }
 }
+
+fn get_signers_epoch(env: &Env, signers_hash: &BytesN<32>) -> Option<u64> {
```

Comment:
```suggestion
fn signers_epoch(env: &Env, signers_hash: &BytesN<32>) -> Option<u64> {
```
`get` is redundant and inconsistent with our naming convention

Reply:
got it. updated.